### PR TITLE
Add systemc

### PR DESCRIPTION
**Pull request by nicmcd**

### CODE REVIEW

**Review by alalazo — APPROVED** [review]

**Review by alalazo — CHANGES_REQUESTED** [review]
There are style issues:
```
==> Error: spack style found errors
  var/spack/repos/builtin/packages/systemc/package.py:10: [E501] line too long (93 > 88 characters)
  var/spack/repos/builtin/packages/systemc/package.py:23: [E111] indentation is not a multiple of four
  var/spack/repos/builtin/packages/systemc/package.py:24: [E111] indentation is not a multiple of four
  var/spack/repos/builtin/packages/systemc/package.py:25: [E121] continuation line under-indented for hanging indent
```

**Comment by nicmcd** [comment]
@alalazo, all good now?

File: var/spack/repos/builtin/packages/systemc/package.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    and more."""
+
+    homepage = "https://www.accellera.org/downloads/standards/systemc"
+    url      = ("https://accellera.org/images/downloads/standards/systemc/"
+                "systemc-2.3.3.tar.gz")
+
+    maintainers = ['nicmcd']
+
+    version('2.3.3', sha256=('5781b9a351e5afedabc37d145e5f7edec08f3fd5de00ffeb8'
+                             'fa1f3086b1f7b3f'))
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/systemc/package.py]: Please don't split the sha (we don't do it in any package).

File: var/spack/repos/builtin/packages/systemc/package.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+from spack import *
+
+
+class Systemc(CMakePackage):
+    """SystemC: System level modeling, design exploration, performance modeling,
+    and more."""
+
+    homepage = "https://www.accellera.org/downloads/standards/systemc"
+    url      = ("https://accellera.org/images/downloads/standards/systemc/"
+                "systemc-2.3.3.tar.gz")
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/systemc/package.py]: Also the url are usually on the same line

**Review by alalazo — CHANGES_REQUESTED** [review]
Two minor style requests, but basically LGTM.

**Comment by nicmcd** [comment]
@alalazo done, PTAL

**Comment by alalazo** [comment]
@spackbot run pipeline

**Review by alalazo — APPROVED** [review]

**Pull request merged**